.prettierrc.yml

```diff
@@ -0,0 +1,7 @@
+semi: true
```

Comment:
It seems like the recomendation from https://github.com/xojs/xo#prettier is to just set 
```
	trailingComma: 'es5',
	singleQuote: false,
	bracketSpacing: true,
```
Not sure about some of the others.

Reply:
Happy to use that instead. Like I said, was trying to find a config that minimizes changes.